packages/expo/CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# @bugnsnag/expo contributing guide
+
+## Keeping dependencies in sync
+
+The Expo notifier depends on some modules whose native code, if it exists, is bundled with Expo core. That means the version we depend on must match, otherwise we get conflicts and/or there are native/JS interface differences.
+
+When a new version of the Expo SDK is released, the dependencies we use must be checked to see if they are up to date.
+
+The following modules are currently used:
+
+- `expo-constants` (`@bugsnag/expo`, `@bugsnag/plugin-expo-app`, `@bugsnag/plugin-expo-device`)
+- `expo-file-system`, `@react-native-community/netinfo` (`@bugsnag/delivery-expo`)
+- `expo-constants` (`@bugsnag/plugin-expo-app`)
+- `expo-device` (`@bugsnag/plugin-expo-device`)
+
+If you add a new dependency please add it to this list.
+
+To check what native module versions are bundled with Expo, check this file:
+
+https://github.com/expo/expo/blob/master/packages/expo/bundledNativeModules.json
+
+To check whan JS-only module versions are depended on by Expo, check this file:
+
+https://github.com/expo/expo/blob/master/changelogVersions.json
+
+## Updating the CLI to install a compatible notifier version
+
+When the version of the bundled native modules changes, making the notifier incompatible with previous Expo SDKs we need to update the CLI using the established pattern. See `packages/expo-cli/commands/install.js` for details.
```

Comment:
I initially read this as "When the version of the bundled native modules changes, the notifier _might_ be incompatible with previous Expo SDKs", when I think it means "When the version of the bundled native modules changes, the notifier _will_ be incompatible with previous Expo SDKs"

Something like this might be clearer?

```suggestion
When the version of the bundled native modules changes the notifier will be incompatible with previous Expo SDKs. To prevent installing the conflicting versions, we need to update the CLI using the established pattern in [`packages/expo-cli/commands/install.js`](../expo-cli/commands/install.js).
```